CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
-# Unreleased
+# Changelog
+All notable changes to this project will be documented in this file.
 
-# v0.1.1
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-* Add option for skipping SSL verification, `CHEF_SKIP_SSL`
+## [Unreleased]
 
-# v0.1.0
-* Initial release
+## [0.1.1] -  2015-12-09
+### Added
+- Add option for skipping SSL verification, `CHEF_SKIP_SSL`
+
+## [0.1.0] - 2015-09-07
+### Added
+- Initial release
```

Comment:
Can add reference pointers for the releases:
```

[Unreleased]: https://github.com/cyberark/summon-conjur/compare/v0.1.1...HEAD
[0.1.1]: https://github.com/cyberark/summon-conjur/compare/v0.1.0...v0.1.1
[0.1.0]: https://github.com/cyberark/summon-conjur/compare/v0.1.0...v0.1.0
```

Reply:
Thanks for getting the links for me